docs/config-file-curations-yml.md

```diff
@@ -86,47 +86,31 @@ Where the list of available options for curations is defined in
 
 ## Command Line
 
-To use the `curations.yml` file put it to `$ORT_CONFIG_DIR/curations.yml` or pass it to the `--package-curations-file`
-option of the _analyzer_:
+To make ORT use the `curations.yml` file, put it to the default location of `$ORT_CONFIG_DIR/curations.yml` and then run
+the _analyzer_:
 
 ```bash
 cli/build/install/ort/bin/ort analyze
   -i [source-code-of-project-dir]
   -o [analyzer-output-dir]
-  --package-curations-file $ORT_CONFIG_DIR/curations.yml
 ```
 
-Alternatively specify a directory with multiple curation files using the `--package-curations-dir` to the _analyzer_:
-
-```bash
-cli/build/install/ort/bin/ort analyze
-  -i [source-code-of-project-dir]
-  -o [analyzer-output-dir]
-  --package-curations-dir $ORT_CONFIG_DIR/curations
-``` 
-
-ORT can use [ClearlyDefined](https://clearlydefined.io/) as a source for curated metadata. The preferred workflow is to
-use curations from ClearlyDefined, and to submit curations there. However, this is not always possible, for example in
-case of curations for organization internal packages. To support this workflow, ClearlyDefined can be enabled as the
-single source for curations or in combination with a `curations.yml` with the `--clearly-defined-curations` option of
-the analyzer:  
-
-```bash
-cli/build/install/ort/bin/ort analyze
-  -i [source-code-of-project-dir]
-  -o [analyzer-output-dir]
-  --package-curations-file $ORT_CONFIG_DIR/curations.yml
-  --clearly-defined-curations
-```
+Alternatively to a single file, curations may also be split across multiple files below a directory, by default
+`$ORT_CONFIG_DIR/curations`. Both file and directory package curation providers may also be configured as
```

Comment:
Should we add a comma after "Both"?

Reply:
If so, I believe we'd need another comma after "providers", too. But I'm fine either way.

Reply:
I guess I'll simply drop "Both" as it sounds a bit as if only these two can be configured in `config.yml`.